packages/app/src/cli/models/extensions/function-specifications/order_routing_location_rule.ts

```diff
@@ -0,0 +1,11 @@
+import {createFunctionSpec} from '../functions.js'
+
+const spec = createFunctionSpec({
+  identifier: 'order_routing_location_rule',
+  externalIdentifier: 'order_routing_location_rule',
+  externalName: 'Order routing location rule',
+  gated: true,
```

Comment:
The [PR](https://github.com/Shopify/cli/pull/825/files#diff-b0e909dd71b245fc4d00358a78abb504fddc6bee0d9eb2068c927280ce14971b) where `public` was changed to `gated`